std/string.d

```diff
@@ -2638,6 +2638,15 @@ auto lineSplitter(KeepTerminator keepTerm = KeepTerminator.no, Range)(Range r)
     return LineSplitter!(keepTerm, Range)(r);
 }
 
+///
+@safe pure unittest
+{
+    import std.algorithm.comparison : equal;
+
+    string s = "Hello\nmy\rname\nis";
+    assert(splitLines(s).equal(lineSplitter(s)));
```

Comment:
Yeah. Like I already said, I don't think this is a useful example at all. Going by this example alone, it just begs the question "so what's the difference?".